### DISCUSSION TITLE
How to bulk configure Stream Proxy?

### DISCUSSION THREAD

**Discussion by zhangdongdong7** [apache/apisix#5652] [Q&A]
官方文档说的，How to enable stream proxy？
apisix:
  stream_proxy: # TCP/UDP proxy
    tcp: # TCP proxy address list
      - 9100
      - "127.0.0.1:9101"
    udp: # UDP proxy address list
      - 9200
      - "127.0.0.1:9211"
配置完成之后才去设置route，如果我有200个tcp端口，应该怎么优雅的实践呢？

**Comment by tokers** [comment]
It seems that we don't support configuring a continuous port range. Now the way to configure such huge port numbers is use the template for `config.yaml` and render it according to your needs.

**Reply by chenwenjun-github** [reply]
> It seems that we don't support configuring a continuous port range. Now the way to configure such huge port numbers is use the template for `config.yaml` and render it according to your needs.

@tokers 
If my port is not fixed for the time being, I want to add a new port every time and load it dynamically without restarting apisix. Can it be done? now apifix is install in k8s, config.yaml is as a configmap of k8s, even if configmap is resync to pod, when I exec 'apifix reload' in pod, it doesn't work, when i exec 'apifix stop' in pod, pod crashoff immediately

**Reply by tokers** [reply]
No, dynamic port listening is lacking.
